src/Runner.jl

```diff
@@ -939,7 +935,7 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
         # Default mappings for some tools
         "CC" => "cc",
         "CXX" => "c++",
-        "OBJC" => "objc",
+        "OBJC" => "cc -x objective-c",
```

Comment:
This is the only thing I'm not 100% sure about: it looks like the `OBJC` environment variable would be useful for FFMPEG (see https://github.com/JuliaIO/FFMPEGBuilder/pull/5 and https://github.com/JuliaPackaging/Yggdrasil/blob/8be19820afa24ff817185f3c53c4d20786a8add3/F/FFMPEG/common.jl#L77), but I'm not sure it's used anywhere else, so it may not be worth keeping it.

Reply:
FFMPEG uses `OBJCC`, not `OBJC` I think; so my guess is that we're fine just dropping this environment variable completely.

Reply:
Strictly speaking, FFMPEG doesn't use any environment variable, but it may be convenient to have one just in case.  In https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/426#issuecomment-526486197 I observed that I couldn't find a really standard environment variable for Objective-C, people seem to use different alternatives, even though `OBJC` seemed the slightly more common one.

Reply:
Yeah, I think it's probably better to just not have an environment variable at all then.